Packs/AzureStorageContainer/Integrations/AzureStorageContainer/AzureStorageContainer.py

```diff
@@ -850,6 +863,152 @@ def set_blob_properties_command(client: Client, args: Dict[str, Any]) -> Command
+        expiry : Expiry time for sas token.
+    Returns:
+        sas token
+
+    """
+    if sip:
+        StringToSign = (sp + "\n" +
+                        signedstart + "\n"
+                        + expiry + "\n"
+                        + cr + "\n"
+                        + "" + "\n"
+                        + sip + "\n"
+                        + "https" + "\n"
+                        + api_version + "\n"
+                        + sr + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "").encode('UTF-8')
+        signed_hmac_sha256 = hmac.new(base64.b64decode(account_key), StringToSign, hashlib.sha256)
+        sig = base64.b64encode(signed_hmac_sha256.digest())
+
+        token = {
+            'sp': sp,
+            'st': signedstart,
+            'se': expiry,
+            'sip': sip,
+            'spr': "https",
+            'sv': api_version,
+            'sr': sr,
+            'sig': sig
+        }
+    else:
+        StringToSign = (sp + "\n" +
+                        signedstart + "\n"
+                        + expiry + "\n"
+                        + cr + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "https" + "\n"
+                        + api_version + "\n"
+                        + sr + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "" + "\n"
+                        + "").encode('UTF-8')
+        signed_hmac_sha256 = hmac.new(base64.b64decode(account_key), StringToSign, hashlib.sha256)
+        sig = base64.b64encode(signed_hmac_sha256.digest())
+        token = {
+            'sp': sp,
+            'st': signedstart,
+            'se': expiry,
+            'spr': "https",
+            'sv': api_version,
+            'sr': sr,
+            'sig': sig
+        }
+
+    sas_token = urllib.parse.urlencode(token)
+    return sas_token
+
+
+def check_valid_permission(valid_permissions: str, input_permissions: str):
+    """
+    Check the permissions follows valid permission order.
+
+    Args:
+        valid_permissions : valid permissions order
+        input_permissions : permissions given
+
+    Returns:
+        bool
+
+    """
+    permissions_length = len(input_permissions)
+    if len(valid_permissions) < permissions_length:
+        return False
+    for i in range(permissions_length - 1):
+        x = input_permissions[i]
+        y = input_permissions[i + 1]
+        last = valid_permissions.rindex(x)
+        first = valid_permissions.index(y)
+        if last == -1 or first == -1 or last > first:
+            return False
+    return True
+
+
+def generate_sas_token(client: Client, args: dict) -> CommandResults:
```

Comment:
```suggestion
def generate_sas_token_command(client: Client, args: dict) -> CommandResults:
```

Please change the function name to keep it in accordance to the other command functions